data/config/tableOneHelper.json

```diff
@@ -0,0 +1,148 @@
+
+{
+    "Subject":[
+        {
+            "name":"Sex",
+            "label":"sex",
```

Comment:
retrieve this from dictionary and gitops instead of duplicating data in this json file